node-server/index.js

```diff
@@ -27,73 +25,32 @@ const getPortArg = () => {
 logger.info('CHATSERVER node starting')
 logger.info('------------------------')
 
-const init = async () => {
-  try {
-    const portArg = getPortArg()
-    await nodeState.initialize(portArg)
-    websocketService.initialize(
-      nodeState.getListenPort(),
-      nodeState.getOtherActiveNodes()
-    )
-
-    logger.info('Node initialized')
-    logger.info('----------------')
-    logger.info(`Listening for WS connection on PORT ${nodeState.getListenPort()}`)
-    logger.info(`Other nodes online: ${nodeState.getOtherActiveNodes()}`)
-    logger.info('----------------')
-  } catch (err) {
-    logger.error('initializing:', err)
-    process.exit(70) // sysexits.h EX_SOFTWARE (internal software error)
-  }
-}
-
-const terminate = async () => {
-  websocketService.terminate()
-  try {
-    await nodeState.close()
-    logger.info()
-    logger.info('------------------')
-    logger.info('CHATSERVER node is terminating')
-    logger.info(`Other nodes running: ${nodeState.getOtherActiveNodes()}`)
-  } catch (err) {
-    logger.error('terminating:', err)
-  }
-}
-//temporarily comment this because it breaks something
-/*
 const run = () => {
   rl.question('input: ', async (input) => {
     switch (input) {
       case 'quit':
         rl.close()
-        await terminate()
+        await app.terminate()
         break
       case 'broadcast':
-        websocketService.broadcastMessageToAll(`MESSAGE to all nodes: I'm listening on port: ${nodeState.getListenPort()}`)
+        app.broadcastMessageToAll(`MESSAGE to all nodes: I'm listening on port: ${nodeState.getListenPort()}`)
         run()
         break
       case 'peers':
         logger.info('Peers:')
-        logger.info('\tOutbound:', websocketService.openOutboundConnections())
-        logger.info('\tInbound:', websocketService.openInboundConnections())
+        logger.info('\tConnections:', app.returnConnections())
```

Comment:
Separating the inbound and outbound connections (as it was implemented before this PR) would be good for testing? Especially considering that client connections will also be added here